src/bitcoin/bitcoinCoreRpc.ts

```diff
@@ -103,7 +103,7 @@ export class BitcoinCoreRpc implements BitcoinBlockchain {
       scanobjects
     );
     if (!result || !result.success || result.unspents === undefined) {
-      logger.error(`Transaction scan failed: ${result}`);
+      logger.crit(`Transaction scan failed: `, result);
```

Comment:
This change and the one above are not uniform.  One uses `%` for string substitution and one does not.  Is there a reason for this?